jquery.canvaswrapper.js

```diff
@@ -500,10 +467,12 @@
                 }
             }
         } else {
-            positions = this.getTextInfo(layer, text, font, angle).positions;
+            info = this.getTextInfo(layer, text, font, angle);
+            positions = info.positions;
             for (i = 0; positions[i]; i++) {
                 position = positions[i];
-                if (position.x === x && position.y === y && position.text === text) {
+                htmlYCoord = y + 0.75 * info.height;
```

Comment:
0.75 :(
How bad does the text look when this coefficient is not used. Basically 1 instead of 0.75?

Reply:
It has a pretty big impact actually and the misalignment is visible :(